pages/common/duplicity.md

```diff
@@ -0,0 +1,23 @@
+# duplicity
+
+> Creates incremental, compressed, encrypted, and versioned backups and optionally uploads them to a variety of backend services.
+
+- Backup a folder via ftps to a remote machine, encrypting with a password:
+
+`FTP_PASSWORD={{ftp_login_password}} PASSPHRASE={{encryption_password}} duplicity {{path/to/source/directory}} {{ftps://user@remotehost.tld/target/directory/path/}}`
```

Comment:
Can we do without the environment variables ? They are cluttering the example and seem to be ftps specific rather than duplicity specific.

Also, remotehost.tld => hostname is slightly better.